app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """
+    """
```

Comment:
The docstring for `NotVaccinatedError` is empty. Consider adding a brief description of when this exception should be raised.